packages/fhir-converter/src/templates/cda/Resources/MedicationStatement.hbs

```diff
@@ -19,7 +19,7 @@
         "dosage":
         [
             {
-                "text":"{{medicationStatement.text._}}",
+                "text":"{{{ parseReferenceData medicationStatement.text._ }}}",
```

Comment:
I looked through all templates and could only find two other places where we are pulling reference data into double brackets such that we would get &quot in json (for example).